Error compiling generated code

**Error:**

```js
$ zenstack generate
⌛️ ZenStack CLI v2.14.2, running plugins
✔ Generating Prisma schema
✔ Generating PrismaClient enhancer
✔ Generating Zod schemas
Error compiling generated code:
node_modules/bun-types/bun.d.ts:6977:5 - error TS1139: Type parameter declaration expected.

6977     const In extends SpawnOptions.Writable = "ignore",
         ~~~~~
node_modules/bun-types/bun.d.ts:6977:14 - error TS1005: ',' expected.

6977     const In extends SpawnOptions.Writable = "ignore",
                  ~~~~~~~
node_modules/bun-types/bun.d.ts:6977:34 - error TS1005: ',' expected.

6977     const In extends SpawnOptions.Writable = "ignore",
                                      ~
node_modules/bun-types/bun.d.ts:6978:15 - error TS1005: ',' expected.

6978     const Out extends SpawnOptions.Readable = "pipe",
                   ~~~~~~~
node_modules/bun-types/bun.d.ts:6978:35 - error TS1005: ',' expected.

6978     const Out extends SpawnOptions.Readable = "pipe",
                                       ~
node_modules/bun-types/bun.d.ts:6979:15 - error TS1005: ',' expected.

6979     const Err extends SpawnOptions.Readable = "inherit",
                   ~~~~~~~
node_modules/bun-types/bun.d.ts:6979:35 - error TS1005: ',' expected.

6979     const Err extends SpawnOptions.Readable = "inherit",
                                       ~
node_modules/bun-types/bun.d.ts:6980:3 - error TS1109: Expression expected.

6980   >(
       ~
node_modules/bun-types/bun.d.ts:6981:12 - error TS1005: ')' expected.

6981     options: SpawnOptions.OptionsObject<In, Out, Err> & {
                ~
node_modules/bun-types/bun.d.ts:6996:19 - error TS1011: An element access expression should take an argument.

6996       cmd: string[]; // to support dynamically constructed commands
    

: Error compiling generated code
```

**package.json:**

```json
{
  "name": "prisma",
  "type": "module",
  "private": true,
  "devDependencies": {
    "@types/bun": "1.2.13",
    "prisma": "6.8.2",
    "zenstack": "2.14.2"
  },
  "dependencies": {
    "@prisma/client": "6.8.2",
    "@zenstackhq/runtime": "2.14.2"
  },
  "devDependencies": {
    "typescript": "5.8.3"
  }
}
```

--- a/packages/schema/src/utils/pkg-utils.ts
+++ b/packages/schema/src/utils/pkg-utils.ts
@@ -67,7 +67,7 @@ export function findNodeModulesFile(name: string, cwd: string = process.cwd()) {
 }
 
 export function getPackageManager(searchStartPath = '.') {
-    const lockFile = findUp(['yarn.lock', 'pnpm-lock.yaml', 'package-lock.json'], searchStartPath);
+    const lockFile = findUp(['yarn.lock', 'bun.lock', 'pnpm-lock.yaml', 'package-lock.json'], searchStartPath);
 
     if (!lockFile) {
         // default use npm
@@ -77,6 +77,7 @@ export function getPackageManager(searchStartPath = '.') {
     const packageManager = match(path.basename(lockFile))
         .with('yarn.lock', () => 'yarn')
         .with('pnpm-lock.yaml', () => 'pnpm')
+        .with('bun.lock', () => 'bun')
         .otherwise(() => 'npm');
 
     return { packageManager, lockFile, projectRoot: path.dirname(lockFile) };
@@ -96,6 +97,14 @@ export function installPackage(
         case 'yarn':
             execSync(
                 `yarn --cwd "${projectPath}" add ${exactVersion ? '--exact' : ''} ${pkg}@${tag} ${dev ? ' --dev' : ''}`
+            );
+            break;
+
+        case 'bun':
+            execSync(
+                `bun install --cwd "${projectPath}" ${exactVersion ? '--exact' : ''} ${
+                    dev ? ' --dev' : ''
+                } ${pkg}@${tag}`
             );
             break;
 

--- a/packages/sdk/src/code-gen.ts
+++ b/packages/sdk/src/code-gen.ts
@@ -15,6 +15,8 @@ export function createProject(options?: CompilerOptions) {
             skipLibCheck: true,
             noEmitOnError: true,
             noImplicitAny: false,
+            skipDefaultLibCheck: true,
+            types: [],
             ...options,
         },
     });

